Remotion/Data/DomainObjects/Infrastructure/DomainObjectTransactionContext.cs

```diff
@@ -21,48 +21,37 @@
 namespace Remotion.Data.DomainObjects.Infrastructure
 {
   /// <summary>
-  /// Provides the default implementation of the <see cref="IDomainObjectTransactionContext"/> interface.
+  /// Provides actual implementation for the <see cref="DomainObjectTransactionContextStruct"/>.
   /// Represents the context of a <see cref="DomainObject"/> that is associated with a specific <see cref="ClientTransaction"/>.
   /// </summary>
-  public class DomainObjectTransactionContext : IDomainObjectTransactionContext
+  public class DomainObjectTransactionContext
```

Comment:
Depending on the definition of public API we might need XML docs and arg checks in this file. But that is something that mike should decide once we have all the other stuff in order.